### DISCUSSION TITLE
Hotkey to mute specific programm

### DISCUSSION THREAD

**Discussion by rwxOsu** [File-New-Project/EarTrumpet#1236] [Ideas]
It would be really nice to be able to set a hotkey to mute a specific program or set it to a predefined volume

**Comment by kanjieater** [comment]
I was going to ask for this as well. A hotkey to mute the program in focus would be very helpful.

Looks like it is already supported here: https://github.com/File-New-Project/EarTrumpet/discussions/870

**Reply by rwxOsu** [reply]
thank you, this works perfectly
